File: src/BaseAdvertising.java
public class BaseAdvertising {
    private int id;
    private int clicks;
    private int views;

    public BaseAdvertising(int id) {
        this.id = id;
    }

    public BaseAdvertising() {
    }

    public void incViews() {
        this.views++;
    }
    public void incClicks() {
        this.clicks++;
    }

    public int getClicks() {
        return clicks;
    }

    public int getViews() {
        return views;
    }
    public String describeMe() {
        return "I am BaseAdvertising class :)";
    }
}

File: python/base_advertising.py

class BaseAdvertising:
    def __init__(self, id=-1):
        self.__id = id
        self.__clicks = 0
        self.__views = 0

    def inc_views(self):
        self.__views += 1

    def inc_clicks(self):
        self.__clicks += 1

    def get_clicks(self):
        return self.__clicks

    def get_views(self):
        return self.__views

    @staticmethod
    def describe_me():
        return "I am BaseAdvertising class :)"


